docs/tutorials/11_deuteron_binding_energy.ipynb

```diff
@@ -0,0 +1,1374 @@
+    "        if n == m:\n",
+    "            return 1\n",
+    "        else:\n",
+    "            return 0\n",
+    "    else:\n",
+    "        raise ValueError(\n",
+    "            \"The function expects two integer numbers ('int') as input, but were given a {} type and a {} type as input.\".format(\n",
+    "                type(n), type(m)\n",
+    "            )\n",
+    "        )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "553b3feb",
+   "metadata": {},
+   "source": [
+    "In [1] we can find the following expression for the Deuteron Hamiltonian\n",
+    "$H_N = \\sum_{n,n'=0}^{N-1}\\langle n'|(T+V)|n\\rangle a_{n'}^\\dagger a_n$,\n",
+    "where $|n\\rangle$ and $|n'\\rangle$ represent quantum states in the harmonic oscillator basis. With that in mind, the operators $a_n^\\dagger$ and $a_n$ represent the operators of creation and annihilation.\n",
+    "\n",
+    "To create the code that defines $H_N$, we need the matrix elements of the kinetic and potential energy. These expressions we can find in [1] written in the following way\n",
+    "$\\langle n'|T|n\\rangle = \\frac{\\hbar\\omega}{2}\\left[\\left(2n+\\frac{3}{2}\\right)\\delta_{nn'}-\\sqrt{n(n+\\frac{1}{2})}\\delta_{nn'+1}-\\sqrt{(n+1)(n+\\frac{3}{2})}\\delta_{nn'-1}\\right],$\n",
+    "$\\langle n'|V|n\\rangle = V_0\\delta_{n0}\\delta_{nn'}.$\n",
+    "\n",
+    "Where $V_0 = -5.68658111 \\ \\textrm{MeV}$ and $\\hbar\\omega = 7 \\ \\textrm{MeV}$. However, the Hamiltonian written in this way cannot be processed directly in a quantum computer, because a quantum computer manipulates qubits through gates based on Pauli matrices. Thus, we need to convert the operators of creation and annihilation in Pauli operators, to do that we need to make use of the Jordan-Wigner transformation\n",
+    "\n",
+    "$a_n^\\dagger \\ \\rightarrow \\ \\frac{1}{2}\\left[\\prod_{j=0}^{n-1}-Z_j\\right](X_n-iY_n),$\n",
+    "\n",
+    "$a_n \\ \\rightarrow \\ \\frac{1}{2}\\left[\\prod_{j=0}^{n-1}-Z_j\\right](X_n+iY_n).$\n",
+    "\n",
+    "Fortunately, in Qiskit Nature we have tools to define fermionic operators and to convert this kind of operators into Pauli operators through the Jordan-Wigner transformation. Initially, we have to define an empty variable called hamiltonian, using the zero() method of the object FermionicOp. After this, we use a for loop to create the string label that is used to create the FermionicOp that we will use in matrix elements of kinetic and potential energies. Then, we use each matrix element to update our initial hamiltonian variable. At the end of the for loop, we have in hands a Hamiltonian in terms of the creation and annihilation operators, so we need to use JordanWignerMapper() and the QubitConverter() to complete the task of obtain the Deuteron Hamiltonian in terms of Pauli operators. In order to obtain more information about the Qiskit Nature tools, we recommend a reading of the Qiskit documentation [3]. "
```

Comment:
Code references should be typeset in monospace (i.e. using backticks "`" in
markdown).